CodeStarterCamp_Week1/main.swift

```diff
@@ -8,5 +8,16 @@
 
 import Foundation
 
-print("Hello, World!")
+func drawIceCream(){
```

Comment:
함수명 동사형으로 잘 작성해주셨네요!

추가로 살짝 코멘트 드리면, 함수 선언과 { 은 한 칸 띄워주시는 게 일반적인 컨벤션입니다. 🙂 
```suggestion
func drawIceCream() {
```